projects/storefrontlib/shared/services/import-export/file-download.service.ts

```diff
@@ -0,0 +1,23 @@
+import { Injectable } from '@angular/core';
+import { ExportFileOptions } from '../../models/file';
+
+@Injectable({
+  providedIn: 'root',
+})
+export class FileDownloadService {
+  download(fileContent: string, fileOptions: ExportFileOptions): void {
+    const { fileName, type, extension } = fileOptions;
+    const blob = new Blob(['\ufeff' + fileContent], {
```

Comment:
Did you double check if the BOM `'\ufeff'` is needed for any charset, or only UTF? How would this interfere with the `charset` set in the mime type?